src/main/java/io/pinecone/PineconeClientConfig.java

```diff
@@ -17,9 +19,6 @@ public class PineconeClientConfig {
      */
     private String apiKey;
 
-    /**
-     * Required project name
-     */
     private String projectName;
 
     private String environment;
```

Comment:
**nit:** Just curious why we're removing the comment here. Should we maybe add one for `environment`?

Reply:
This was supposed to removed for a previous PR but we made projectName and environment optional if connection url is present.